src/components/Bookmark.spec.js

```diff
@@ -0,0 +1,51 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import Bookmark from './Bookmark'
+
+describe('Bookmark', () => {
+  it('has a bookmark image', () => {
+    render(<Bookmark />)
+
+    const img = screen.getByRole('img')
+
+    expect(img).toBeInTheDocument()
+  })
+
+  it('has an alt attribute', () => {
+    render(<Bookmark />)
+
+    const alt = screen.getByAltText('bookmark')
```

Comment:
Depending on whether you change the alt text according to my thoughts you may need to adjust your test here to check whether an alt attribute is present (but with no explicit value)

Reply:
Same 👍